src/components/FiltersBundle/types.ts

```diff
@@ -18,6 +18,13 @@ export interface ResetFilter {
 
 export interface SearchFilter {
   onChange: (value: string) => void;
+  value?: string;
+  inputRef?: React.RefObject<HTMLInputElement>;
```

Comment:
Why do we need a ref here? Everything related to input changes should be handled using the `value` and `onChange` properties

Reply:
done